_lib/core/common.php

```diff
@@ -573,7 +576,7 @@ function send_mail($opts = []): bool
         
         return $response;
     } elseif (class_exists('PHPMailer\PHPMailer\PHPMailer')) {
-        $mail = new PHPMailer\PHPMailer\PHPMailer();
+        $mail = new PHPMailer();
```

Comment:
this will break things as I removed the namespace

Reply:
I think I've restored this?